canton/BUILD.bazel

```
@@ -319,6 +319,7 @@ scala_library(
         "//canton:community_util-logging",
         "//canton:daml-common-staging_daml-errors",
         "//daml-lf/data",
+        "//daml-lf/language:language",
```

`./fmt.sh --diff` says

```suggestion
        "//daml-lf/language",
```